### ISSUE TITLE
[A11y] Contrasts partly seem not to be WCAG-compliant (AAA & AA)

### ISSUE THREAD

**Issue by mariohamann** [corona-warn-app/cwa-documentation#195]
Also described in: https://github.com/corona-warn-app/cwa-app-ios/issues/229

## Current Implementation
I have analyzed the [High-Resolution Screenshots](https://github.com/corona-warn-app/cwa-documentation/blob/master/ui_screens.md) with the [WebAIM Contrast-Checker](https://webaim.org/resources/contrastchecker/) I found that some of the contrasts do not reach WCAG AAA, sometimes not even WCAG AA.

(The following image may show other colours because of web compression.)
![contrasts](https://user-images.githubusercontent.com/26542182/83799430-12e4fc00-a6a6-11ea-996e-290ae74b17b3.jpg)

Screen 1 (start screen): Both the green area and the blue button do not offer enough contrast. In addition, the perception of the color "blue" decreases most with age ([Source](https://www.smashingmagazine.com/2015/02/designing-digital-technology-for-the-elderly/)), which is why the lower button in combination with the continuous capitalization and the narrow font could cause problems for certain user groups. 

Screen 2 ("Low risk") and 3 ("Risk determination") both contain a small grey font. This is just above the AA minimum of 4.5 from a contrast perspective, but in combination with the very small font could be very difficult to read for some user groups. 

Screen 3 ("Risk assessment"): The switch may not be recognizable as such for some user groups - some may just see a dot.
## Suggested Enhancement
For reasons of accessibility, the contrasts should be increased. At the same time, the difference between headings and "secondary text" must of course be maintained. It might be advisable to create a stronger distinction here by using appropriate font styles.

## Expected Benefits
The application should be easy to use for all user groups - but individual risk groups such as senior citizens are very likely to have higher usability requirements, also in terms of contrasts. In order not to exclude these user groups from using the application, WCAG AAA should be implemented in the best possible way.

**Comment by SebastianWolf-SAP** [comment]
@mariohamann Thank you very much for taking the time to look into these screens and provide us feedback! We highly appreciate that and take it into close consideration.
 
On the points stated: 
1.	The foreground-background contrast ratios for all texts through the UI are strictly monitored in our design work as a key compliance factor. For that reason, we can assure that these meet the WCAG AA for a minimum of 4.5:1. 
A potential reason for the differing values on your end might be due to PNG compression in the screen exports or another technical issue. (i.e. For the texts in the sated cases we are using pure white #FFFFFF).
2.	Typography-wise we are abiding by the default font styles in accordance with Material Design’s guidelines. We make use of the accessibility feature for dynamic font sizes, which gives the user the option to adapt the font size based on their vision’s needs and in accordance with the device’s native Accessibility settings. 
3.	As with the majority of the native controls through the Corona-Warn-App’s interface, we are making use of native controls as much as possible, which also includes the switch control. There, we are simply reusing what the user already has as an existing and familiar control on their Android device of choice. 

Mit freundlichen Grüßen/Best regards, 
SW
Corona Warn-App Open Source Team
